src/components/List/ListItem/ListItem.test.jsx

```diff
@@ -1,59 +1,73 @@
 import React from 'react';
 import { render, fireEvent, screen } from '@testing-library/react';
 import { Add16, Edit16 } from '@carbon/icons-react';
+import TestBackend from 'react-dnd-test-backend';
+import { DragDropContext } from 'react-dnd';
 
 import { Tag } from '../../Tag';
 
-import ListItem from './ListItem';
+import { UnconnectedListItem } from './ListItem';
+
+const wrapInTestContext = (DecoratedComponent, props) =>
+  DragDropContext(TestBackend)(() => <DecoratedComponent {...props} />);
```

Comment:
Should this be in a beforeEach?

Reply:
We have another issue opened that we can address the testing comments.  #1607